Source/WebCore/xml/DOMParser.cpp

```diff
@@ -46,7 +46,7 @@ Ref<DOMParser> DOMParser::create(Document& contextDocument)
 
 ExceptionOr<Ref<Document>> DOMParser::parseFromString(Variant<RefPtr<TrustedHTML>, String>&& string, const AtomString& contentType)
 {
-    auto stringValueHolder = trustedTypeCompliantString(*protectedContextDocument()->protectedScriptExecutionContext().get(), WTFMove(string), "DOMParser parseFromString"_s);
+    auto stringValueHolder = trustedTypeCompliantString(*protectedContextDocument().get(), WTFMove(string), "DOMParser parseFromString"_s);
```

Comment:
Does it work without `get()`? I think the `*` operator works on `RefPtr` directly.
```suggestion
    auto stringValueHolder = trustedTypeCompliantString(*protectedContextDocument(), WTFMove(string), "DOMParser parseFromString"_s);
```

Reply:
Good point.